src/session_pool/events.js

```diff
@@ -0,0 +1,3 @@
+export default {
+    DISCARDED: 'discarded',
```

Comment:
`discarded` => `sessionDiscarded` ... or shouldn't it be `sessionRetired`, since it's called inside `retire()` function